### ISSUE TITLE
babel recipe build fail

### ISSUE THREAD

**Issue by RobertFlatt** [kivy/python-for-android#2803]
### Logs

```
[1m[INFO][0m:    Installing babel into site-packages
[1m[INFO][0m:    [36m-> directory context /home/bobf/ex/regression/.buildozer/android/platform/build-arm64-v8a/build/other_builds/babel/arm64-v8a__ndk_target_21/babel[39m
[1m[90m[DEBUG][39m[0m:   [90m->[0m running python3 setup.py install -O2 --root=/home/bobf/ex/regression/.buildozer/android/platform/build-arm64-v8a/build/python-installs/myapp/arm64-v8a --install-lib=.[0m
[1m[90m[DEBUG][39m[0m:   	Traceback (most recent call last):
[1m[90m[DEBUG][39m[0m:   	  File "/home/bobf/ex/regression/.buildozer/android/platform/build-arm64-v8a/build/other_builds/babel/arm64-v8a__ndk_target_21/babel/setup.py", line 13, in <module>
[1m[90m[DEBUG][39m[0m:   	    from babel import __version__
[1m[90m[DEBUG][39m[0m:   	  File "/home/bobf/ex/regression/.buildozer/android/platform/build-arm64-v8a/build/other_builds/babel/arm64-v8a__ndk_target_21/babel/babel/__init__.py", line 20, in <module>
[1m[90m[DEBUG][39m[0m:   	    from babel.core import UnknownLocaleError, Locale, default_locale, \
[1m[90m[DEBUG][39m[0m:   	  File "/home/bobf/ex/regression/.buildozer/android/platform/build-arm64-v8a/build/other_builds/babel/arm64-v8a__ndk_target_21/babel/babel/core.py", line 14, in <module>
[1m[90m[DEBUG][39m[0m:   	    from babel import localedata
[1m[90m[DEBUG][39m[0m:   	  File "/home/bobf/ex/regression/.buildozer/android/platform/build-arm64-v8a/build/other_builds/babel/arm64-v8a__ndk_target_21/babel/babel/localedata.py", line 17, in <module>
[1m[90m[DEBUG][39m[0m:   	    from collections import MutableMapping
[1m[90m[DEBUG][39m[0m:   	ImportError: cannot import name 'MutableMapping' from 'collections' (/home/bobf/ex/regression/.buildozer/android/platform/build-arm64-v8a/build/other_builds/hostpython3/desktop/hostpython3/Lib/collections/__init__.py)

```

**Comment by RobertFlatt** [comment]
I really don't know what people use for this. 
I do know that nobody is using Babel on Android because it broken.
Go ahead kill it, worst case we have to bring it back.

**Comment by misl6** [comment]
Even if may not be the lightweight solution, I use that to format dates based on the user locale on Android and iOS apps.

Unfortunately, I never encountered the build issue, since I pinned a specific version since the introduction on my codebase.

> Go ahead kill it, worst case we have to bring it back.

When it comes to cleaning code, I do not think twice.

**Comment by misl6** [comment]
Fixed via #2826
